contrib/labgrid-webapp

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+import argparse
+import logging
+import os
+import sys
+
+import graphviz
+import uvicorn
+
+from fastapi import FastAPI
+from fastapi.responses import Response
+
+from labgrid.remote.client import ClientSession, start_session
+from labgrid.remote.common import Place
+from labgrid.resource import Resource
+from labgrid.util.proxy import proxymanager
+
+
+async def do_graph(session: ClientSession) -> bytes:
+    '''Generate a graphviz graph of the current configuration.
+
+    Graph displays:
+    - all resources, grouped by groupname and exporter.
+    - all places, with a list of tags
+    - solid edges between places and acquired resources
+    - dotted edges between places and unacquired resources
+    - edges between resources and places carry the match name if any.
+    '''
+    def res_node_attr(name: str, resource: Resource) -> dict[str, str]:
```

Comment:
I got the following error when running the code:

>   File "./contrib/labgrid-webapp", line 180, in get_graph
>     svg = await do_graph(session)
>   File "./contrib/labgrid-webapp", line 35, in do_graph
>     def res_node_attr(name: str, resource: Resource) -> dict[str, str]:
> TypeError: 'type' object is not subscriptable


Adding single quotes around the type annotation resolved it.
`def res_node_attr(name: str, resource: Resource) -> 'dict[str, str]':`

Reply:
This comes from using different versions of Python. I'll update to use `typing.Dict` instead